Library/Homebrew/missing_formula.rb

```diff
@@ -0,0 +1,106 @@
+require "formulary"
+require "tap"
+require "utils"
+
+module Homebrew
+  module MissingFormula
+    class << self
```

Comment:
Sorry, not `extend`, the `require` and using macOS specific stuff at the bottom of the file.